gnd/src/main/java/com/google/android/gnd/ui/home/HomeScreenFragment.java

```diff
@@ -120,6 +130,14 @@ public void onCreate(@Nullable Bundle savedInstanceState) {
     viewModel.getShowAddFeatureDialogRequests().observe(this, this::onShowAddFeatureDialogRequest);
     viewModel.getFeatureSheetState().observe(this, this::onFeatureSheetStateChange);
     viewModel.getOpenDrawerRequests().observe(this, __ -> openDrawer());
+
+    showFeatureDialogRequests = PublishSubject.create();
+
+    Observable<Feature> featureStream =
+        showFeatureDialogRequests.flatMapMaybe(
+            fragmentManager -> addFeatureDialogFragment.show(fragmentManager));
```

Comment:
This can be simplified as `addFeatureDialogFragment::show)`.

Reply:
Done, thanks!